cluster.tf

```diff
@@ -130,3 +130,25 @@ resource "aws_iam_role_policy_attachment" "cluster_AmazonEKSServicePolicy" {
   policy_arn = "${local.policy_arn_prefix}/AmazonEKSServicePolicy"
   role       = local.cluster_iam_role_name
 }
+
+/*
+ Adding a policy to cluster IAM role that allow permissions
+ required to create AWSServiceRoleForElasticLoadBalancing service-linked role by EKS during ELB provisioning
+*/
+
+data "aws_iam_policy_document" "cluster_elb_sl_role_creation" {
+  count = var.manage_cluster_iam_resources && var.create_eks ? 1 : 0
+
+  statement {
+    effect    = "Allow"
+    actions   = ["ec2:DescribeAccountAttributes"]
```

Comment:
Hi! 

Ran into this same issue, and ended up needing `ec2:DescribeInternetGateways` in addition to the above IAM role.

Thanks for the fix. :bow:

Reply:
Hi James,

Thank you for the suggestion regarding `ec2:DescribeInternetGateways` permissions that must be attached to the IAM role as well.